config/playground/jobs/sample_data_tooling/refresh_deployment.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: refresh-data-cronjob
+spec:
+  # Runs everyday at 1 am UTC
+  schedule: "0 1 * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - name: refresh-data-job
+              image: huyaboo/demo
+              env:
+              - name: SAMPLE_DATA_USERNAME
+                valueFrom:
+                  secretKeyRef:
+                    name: demo-secret
+                    key: username
+              - name: SAMPLE_DATA_PASSWORD
+                valueFrom:
+                  secretKeyRef:
+                    name: demo-secret
+                    key: password
+              - name: HOST
```

Comment:
what's the Host value here? do you have doc for how to apply this file ? thanks

Reply:
`HOST` has been changed to `SAMPLE_DATA_HOST` with an example value. As for the doc, I will make another PR for that